src/js/lib/safe.js

```diff
@@ -159,26 +166,47 @@ Pebble.sendAppMessage = function(message, success, failure) {
 /* Wrap setTimeout and setInterval */
 var originalSetTimeout = setTimeout;
 window.setTimeout = function(callback, delay) {
+  if (safe.warnSetTimeoutNotFunction !== false && typeof callback !== 'function') {
```

Comment:
What is the default JS behavior? Why do we need to do this?

Error message looks fine to me. I am just surprised that we have to do this.